examples/tree_1d_dgsem/elixir_shallowwater_beach.jl

```diff
@@ -0,0 +1,107 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations1D(gravity_constant=9.812)
+cfl = 0.2
```

Comment:
```suggestion
```
This is a strange place to set the `cfl` value. I would move it closer to the `stepsize_callback` creation. Also, since you are using error based time stepping, there is no CFL number needed. so this could be removed.